From c98a9dd4d0a0e4dc9d09b2a7f3ec599f98bf4411 Mon Sep 17 00:00:00 2001
From: Mark Collins <pixelyion@protonmail.com>
Date: Mon, 13 Nov 2023 11:56:16 +0000
Subject: [PATCH libXmu] Ignore test-driver

---
 .gitignore | 1 +
 1 file changed, 1 insertion(+)

diff --git a/.gitignore b/.gitignore
index 0efb7f9..b45e417 100644
--- a/.gitignore
+++ b/.gitignore
@@ -76,3 +76,4 @@ core
 #		Edit the following section as needed
 # For example, !report.pc overrides *.pc. See 'man gitignore'
 #
+test-driver
\ No newline at end of file
-- 
2.42.0

